ccpp/data/GFS_typedefs.F90

```diff
@@ -6506,6 +6549,16 @@ subroutine diag_create (Diag, IM, Model)
     allocate (Diag%tdomzr   (IM))
     allocate (Diag%tdomip   (IM))
     allocate (Diag%tdoms    (IM))
+    allocate (Diag%skebu_wts(IM,Model%levs))
```

Comment:
Why do we need all these variable duplicates in the Intdiag type? I remember that recently @pjpegion cleaned up that duplication. We just need the variables in the `Coupling` type. The standard name for the Intdiag variables contains `_flipped` which is not true - the copy statements in the stochastic physics wrapper clearly do the same for the Coupling and the Intdiag variables.

Please try to remove lines 6552-6561 and everything else that uses those variables.

Reply:
Removed all references to variable duplicates using the Intdiag type and confirmed the variables exist in the Coupling type.